internal/options/kube.go

```diff
@@ -2,16 +2,14 @@ package options
 
 import (
 	"context"
-	"crypto/tls"
-	"crypto/x509"
 	"fmt"
+	"k8s.io/client-go/transport"
```

Comment:
Imports statement is not sorted: should be placed in the underlying block along with `k8s.io/client-go/rest`